src/main/java/org/moltimate/moltimatebackend/service/ActiveSiteService.java

```diff
@@ -26,8 +29,8 @@
 @Slf4j
 public class ActiveSiteService {
 
-    private static final String PROMOL_CSV_PATH = "src/main/resources/motifdata/promol_active_sites.csv";
-    private static final String CSA_CSV_PATH = "src/main/resources/motifdata/csa_curated_data.csv";
+    private static final String PROMOL_CSV_URL = "https://raw.githubusercontent.com/moltimate/moltimate-backend/master/src/main/resources/motifdata/promol_active_sites.csv";
```

Comment:
Can you add a `todo` saying" Refer to the copy of these in the resources folder that should be stored on the cloud device as well